salt/utils/url.py

```diff
@@ -46,8 +46,11 @@ def create(path, saltenv=None):
     path = salt.utils.data.decode(path)
 
     query = f"saltenv={saltenv}" if saltenv else ""
-    url = salt.utils.data.decode(urlunparse(("file", "", path, "", query, "")))
-    return "salt://{}".format(url[len("file:///") :])
+    url = salt.utils.data.decode(urlunparse(("salt", "", path, "", query, "")))
```

Comment:
I am not sure that you can safely just replace "file" with "salt". There is special handling for a `file` scheme; I am not sure the extent of what is different but it could affect how the path and query are handled.

Reply:
ok, reverted to original format with `file:`.

Reply:
I read through the urlunparse source code and there does not appear to be any current special handling of the `file` scheme so it would be safe to replace with `salt` (ignoring future changes that add special cases for file if some bug was discovered)